src/DynamoCoreWpf/Interfaces/IWatchHandler.cs

```diff
@@ -218,9 +218,13 @@ private WatchViewModel ProcessThing(MirrorData data, List<string> preferredDicti
 
         private static string ToString(object obj)
         {
-            return ReferenceEquals(obj, null)
-                ? Resources.NullString
-                : (obj is bool ? obj.ToString().ToLower() : obj.ToString());
+            return ReferenceEquals(obj, null)? 
+                Resources.NullString
+                : (obj is bool ? 
+                    obj.ToString().ToLower()
+                    : obj is double?
+                        ((double)obj).ToString(CultureInfo.InvariantCulture)
+                        : obj.ToString());
```

Comment:
@QilongTang this is a little confusing to me, can you elaborate on what is happening a bit just for my future knowledge.

Reply:
@alfarok @mjkkirschner This code path is not triggerred in `RC1.3.3` when connecting a number node with watch node, but in RC2.0.0 it is to convert its node value from a string to its double value, e.g. string `3.14` in `fr-FR` the value will be `3,14` which will be incorrectly displayed as watch value. This fix here is adding invariant culture translation to it in this case.